src/wasm-type.h

```diff
@@ -148,18 +148,6 @@ std::ostream& operator<<(std::ostream& os, ParamType t);
 std::ostream& operator<<(std::ostream& os, ResultType t);
 std::ostream& operator<<(std::ostream& os, Signature t);
 
-constexpr Type none = Type::none;
-constexpr Type i32 = Type::i32;
-constexpr Type i64 = Type::i64;
-constexpr Type f32 = Type::f32;
-constexpr Type f64 = Type::f64;
-constexpr Type v128 = Type::v128;
-constexpr Type funcref = Type::funcref;
-constexpr Type anyref = Type::anyref;
-constexpr Type nullref = Type::nullref;
-constexpr Type exnref = Type::exnref;
-constexpr Type unreachable = Type::unreachable;
-
```

Comment:
this enforcees that we must use the `Type::` notation, is that right?

Reply:
That's correct. I'll follow up separately on #2572; it isn't materially affected by this PR.